### ISSUE TITLE
Teachingbubble missing required ARIA attribute: aria-level

### ISSUE THREAD

**Issue by mosesyap** [microsoft/fluentui#17177]
### Environment Information

- **Package version(s)**: 7.155.3
- **Browser and OS versions**: All browsers/Win10

### Describe the issue:
Following up on 
https://github.com/microsoft/fluentui/issues/13441

TeachingBubble is still throwing Accessibility Insights error on role="heading" without an aria-level

#### Actual behavior:

TeachingBubble has role="heading" but no "aria-level"

#### Expected behavior:

I think aria-level should be set to 2

**Comment by joschect** [comment]
@mosesyap Thanks for the submission! This looks like a good candidate for a fix. We welcome community fixes, would you have some time to submit something/?

**Comment by seniroberts** [comment]
Commenting on this. Any chance of bubbling this up?
